packages-content-model/roosterjs-content-model-editor/lib/publicApi/utils/paste.ts

```diff
@@ -45,32 +47,32 @@ export default function paste(
         clipboardData.snapshotBeforePaste = editor.getContent(GetContentMode.RawHTMLWithSelection);
     }
 
-    const event = createBeforePasteEvent(
+    const eventData = createBeforePasteEventData(
```

Comment:
Is this needed? The event object should be accesable fine before the change

Reply:
I changed this function to create a object that has only the event data and not the eventType, then we use it to trigger the plugin event and use the return value of triggerPluginEvent that had been the modified by plugin actions to format the model. Without this change, we were using the object created in the paste file and not the one edited by plugins.

Reply:
Let me test some scenarios, since I need to fix the unit test for this function